test/cmdlineTests/evm_to_wasm/output

```diff
@@ -58,7 +58,7 @@ Text representation:
     (local.set $_1 (i64.const 0))
     (call $mstore_internal (i64.const 0) (local.get $_1) (local.get $_1) (local.get $_1) (local.get $_1))
     (call $mstore_internal (i64.const 32) (local.get $_1) (local.get $_1) (local.get $_1) (i64.const 1))
-    (call $eth.storageStore (i32.wrap_i64 (i64.const 0)) (i32.wrap_i64 (i64.const 32)))
+    (call $eth.storageStore (i64.const 0) (i64.const 32))
```

Comment:
This seems to be wrong :(

Reply:
Yeah, it always generates 64-bit consts and the reason is simple - we only have 64-bit literals in `WasmAST.h` :)

```
struct Literal { uint64_t value; };
```